bot/bot.py

```diff
@@ -0,0 +1,20 @@
+import os
+from slackclient import SlackClient
+
+slack_token = os.environ["SLACK_BOT_TOKEN"]
+
+class Bot:
+
+  def __init__(self):
+    self.sc = SlackClient(slack_token)
+
+  def sendMsg(self, message, channel):
```

Comment:
Can you add unit tests for this method? You will probably want to use [`unittest.mock`](https://docs.python.org/3/library/unittest.mock.html) to mock the `SlackClient` (let me know if you want some help looking at that - it may require you to refactor the class a bit to follow [dependency inversion](https://en.wikipedia.org/wiki/Dependency_inversion_principle) better).